.github/workflows/checks.yml

```diff
@@ -46,7 +46,10 @@ jobs:
 
     - name: Run M1 DA Light Node tests in nix environment
       # adjust the log level while debugging
-      run: CELESTIA_LOG_LEVEL=FATAL nix develop --command bash  -c "just suzuka-full-node native build.setup.test.local -t=false"  
+      run: |
+        CELESTIA_LOG_LEVEL=FATAL nix develop --command bash  -c "just suzuka-full-node native build.setup.test.local -t=false"  
+        # see if restarts are graceful
+        CELESTIA_LOG_LEVEL=FATAL nix develop --command bash  -c "just suzuka-full-node native build.setup.test.local -t=false"  
```

Comment:
Yeah, but how else would you test it?

Reply:
It shouldn't be twice the "running" time though because currently a big portion of the "running" time is actually build time. So, we would skip all of that and just start up the network.